src/client/components/github/workflowRun.ts

```diff
@@ -0,0 +1,41 @@
+import { LitElement, html, unsafeCSS } from 'lit'
+import { customElement, property } from 'lit/decorators.js'
+
+@customElement('github-workflow-run')
+export class WorkflowRun extends LitElement {
+
+    @property()
+    status?: string
+
+    @property()
+    runNumber?: string
+
+    @property()
+    htmlUrl?: string
+
+    @property()
+    displayTitle?: string
+
+    @property()
+    avatarUrl?: string
+
+    @property()
+    githubUserName?: string
+
+    static styles = unsafeCSS(require('!!raw-loader!./styles/workflowRun.css').default)
```

Comment:
I don't feel very confident in this approach. The raw-loader project hasn't seen any updates for 3 years and using `require` in an ESM module (as loaded in a browser environment) does not seem to be the right approach.

An alternative on top of my mind is creating e.g. a `workflow.css.ts` file that does something like this:

```ts
export default /*css*/`
  .className { ... }
  // ...
```

which would continue offer us code highlighting and autocompletion but also allow us to simply import the CSS as a string.

Another alternative is to use https://webpack.js.org/loaders/css-loader/.

What do you think?

Reply:
Fixed, for now loading the CSS AS string template literal